deploy/docker/common/nginx-config.conf

```diff
@@ -11,6 +11,16 @@ server {
     gzip_buffers 16 8k;
     gzip_http_version 1.1;
 
+    # to handle uri issue 414 from nginx
+    fastcgi_buffers 8 16k;
+    fastcgi_buffer_size 32k;
+
+    client_max_body_size 24M;
+    client_body_buffer_size 128k;
```

Comment:
Possible values for `client_body_buffer_size` are:
- 8k for 32-bit platforms
- 16k for 64-bit platforms

Reply:
32 and 64 bit platforms comprises of most of the system out there.

But yeah, my bad, 8k and 16k are default values for those platforms.

Nevertheless, I don't think we would need to modify it since it is handled by Nginx itself in case client body buffer exceeds by creating temporary file.

Ref: https://nginx.org/en/docs/http/ngx_http_core_module.html#client_body_buffer_size

Reply:
ok removing